BUILD.md

```diff
@@ -0,0 +1,86 @@
+# Important notice
+***In order to uniformize and simplify the build system we had to make choices. TC is currently only officially supported on Ubuntu 16.04 with gcc 5.4.0, cuda 9.0 and cudnn 7.***
+Other configurations may work too but are not yet officially supported.
+For more information about setting up the config that we use to build the conda dependencies see the following [Dockerfile](conda_recipes/Dockerfile).
+
+Our main goal with this decision is to make the build procedure extremely simple, reproducible both internally and extensible to new targets in the future.
+In particular the gcc-4 / gcc-5 ABI switch is not something we want to concern ourselves with at this point, we go for gcc-5.4.0.
+
+# Conda from scratch (one time config)
+Choose and set a CONDA_PATH then run the following:
```

Comment:
You are using `CONDA_PATH` here, but `CONDA_PREFIX` somewhere below.  Is the difference intended?